versioned_docs/version-22.04/resources/glossary.md

```diff
@@ -0,0 +1,284 @@
+
+A plugin is a monitoring probe, i.e. a binary executable or a script that is called by the [monitoring engine](#monitoring-engine) to carry out a check on a [host](#host) or [service](#service). The plugin determines which status should be sent to the monitoring engine, based on the checks it makes and on the thresholds defined in the configuration of the host or service.
+
+## Plugin Pack
+
+The term "Plugin pack" refers to a [plugin](#plugin) and the corresponding pack.
+
+A pack contains the configuration of the plugin in Centreon (command, [templates](#template), thresholds), as well as data required by the automatic discovery feature.
+
+**See also**:
+
+- [Plugin Packs](../monitoring/pluginpacks.md),
+- [Introduction to Plugin Packs](/pp/integrations/plugin-packs/getting-started/introduction).
+
+## Poller
+
+Centreon server used in a [distributed architecture](#architecture-simple-vs-distributed). A poller can be attached to a [remote server](#remote-server), or directly to a [central server](#central-server).
+
+- A Centreon poller monitors [resources](#resource). It has a [monitoring engine](#monitoring-engine).
+
+- A poller has no graphical interface: the resources it monitors are displayed in the interface of the central server and of the remote server it is attached to.
+
+"Poller" is also used to refer to the monitoring engine that is present in a central server, a remote server and a poller.
+
+## Pull mode
+
+Advanced configuration for Centreon [Gorgone](#gorgone) that reverses the direction of ZMQ flows. The communication is initiated by pollers and remote servers, and is directed to the central server. This mode is commonly used by MSPs.
+
+**See also**: [Configuring Gorgone in pull mode](../developer/developer-gorgone-pull-mode.md).
+
+## Recurring downtime
+
+Recurring downtimes are [downtimes](#downtime) that occur regularly.
+
+**See also**: [Recurrent downtimes](../alerts-notifications/downtimes.md#recurrent-downtimes).
+
+## Remote server
+
+Centreon server used in a [distributed architecture](#architecture-simple-vs-distributed). A remote server is attached to a central server. Pollers can be attached to a remote server.
+
+- A remote server monitors [resources](#resource). It has a [monitoring engine](#monitoring-engine).
+
+- It has a graphical interface, but no configuration menus.
+
+- The resources it monitors are displayed in its interface, and in the interface of the central server it is attached to.
+
+## Resource
+
+Object monitored by a Centreon platform ([hosts](#host), [services](#service), metaservices).
+
+## Retention files
+
+Retention files belong to Centreon [Broker](#broker). These files store the monitoring data that could not be inserted into the database. For instance, if a communication problem occurs between Engine and Broker, the data is not lost, Broker stores them in a file (queue). The file will then be read by Centreon Broker, then inserted into the databases so as to avoid data loss.
```

Comment:
```suggestion
Retention files belong to Centreon [Broker](#broker). These files store the monitoring data that could not be inserted into the database. For instance, if a communication problem occurs between Engine and Broker, the data is not lost, Broker stores them in a file (whose name includes the term "queue"). The file will then be read by Centreon Broker, then inserted into the databases so as to avoid data loss.
```